yarn-project/bb-prover/src/prover/bb_prover.ts

```diff
@@ -640,6 +641,7 @@ export class BBNativeRollupProver implements ServerCircuitProver {
         proofFileName,
         verificationKeyPath!,
         logFunction,
+        verificationFunction instanceof verifyAvmProof ? path.join(bbWorkingDirectory, CALLDATA_FILENAME) : undefined,
```

Comment:
Using instanceof here is too much of a hack:
(1) breaks encapsulation
(2) instanceof in TS should be avoided, it's a major source of bugs

Reply:
With passing calldata as part of the proof, this change was reverted.